packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -285,31 +286,31 @@ export const useStyles = makeStyles(
         '& .MuiDataGrid-columnHeaderWrapper .MuiDataGrid-cell': {
           borderBottom: 'none',
         },
-        '& .MuiDataGrid-cellWithRenderer': {
+        '& .MuiDataGrid-cell--withRenderer': {
           display: 'flex',
           alignItems: 'center',
         },
         '& .MuiDataGrid-withBorder': {
           borderRight: `1px solid ${borderColor}`,
         },
-        '& .MuiDataGrid-cellLeft': {
+        '& .MuiDataGrid-cell--textLeft': {
           textAlign: 'left',
         },
-        '& .MuiDataGrid-cellLeft.MuiDataGrid-cellWithRenderer, & .MuiDataGrid-cellLeft.MuiDataGrid-cellEditing':
+        '& .MuiDataGrid-cell--textLeft.MuiDataGrid-cell--withRenderer, & .MuiDataGrid-cell--textLeft.MuiDataGrid-cell--editing':
           {
             justifyContent: 'flex-start',
           },
-        '& .MuiDataGrid-cellRight': {
+        '& .MuiDataGrid-cell--textRight': {
```

Comment:
should we adopt a convention here to cleanup things a bit?
We have `textRight`, `alignRight`, `right` also I think. Should we pick one?

Reply:
The thing is that these 3 classes do different things, the `--textRight` one does `text-align: right`.

Reply:
I see, I didn't check properly, I saw the next line `justifyContent: 'flex-end',`! 
My bad!
👍